test/jdk/java/lang/IO/IO/IO.java

```diff
@@ -51,7 +51,7 @@
 /*
  * @test
  * @bug 8305457 8342936 8351435
```

Comment:
Issue id needs to be added (and other test files too)

Reply:
Sorry, I missed this comment. Fixed here:
https://github.com/openjdk/jdk/pull/24438/commits/5d02ff6f34e20374627d0ef0b31e9db4e1bf6f2b